website/docs/docs/deploy/job-triggers.md

```diff
@@ -27,3 +61,15 @@ A custom cron schedule can be specified in the Job Settings page when you edit a
 6. Enter the custom cron syntax for the schedule you want.
 
 <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/job-schedule.png" title="Schedule your dbt job"/>
+
+THE BELOW IS JUST COPIED OVER TEXT - WILL AMEND SOON!
```

Comment:
@mirnawong1 this will need some help, I added back in Jobs but not sure what else you had in mind :)